src/pages/home/home.module.scss

```diff
@@ -0,0 +1,53 @@
+body, h1, h2, h3, h4, h5, h6, p, blockquote, pre, dl, dd, ol, ul, figure, hr {
```

Comment:
Hm... Isso aqui n deveria ter sido removido ontem?

To confuso, me lembra de falar disos na daily.